chapters/equations.tex

```diff
@@ -490,7 +442,7 @@ \subsection{assert}\label{assert}
   \lstinline!message! indicates the cause of the warning.
   \begin{nonnormative}
   It is recommended to report the warning only once when the condition becomes false, and it is reported that the condition is no longer
-  violated when the condition returns to true. The \lstinline!assert! statement shall have no influence on the behavior of the model.
+  violated when the condition returns to true.  The \lstinline!assert! statement shall have no influence on the behavior of the model.
```

Comment:
A newly introduced redundant space.

Reply:
Yes, and what a good opportunity to talk about the following way out of line breaking mayhem.

I've seen many authors adopt a line breaking style where physical breaks are only placed after each sentence.  I like this approach very much because of the predictability and stability it gives to the source code:
- Git diffs get localized to the single sentences being changed.
- Changes within a sentence (such as adding missing `\lstinline` formatting) have no impact on nearby sentences.
- No spurious whitespace changes within sentences due to re-flowing text to a fixed width.

A neighborhood of the commented line have been updated according to this style.  Thoughts?

Reply:
Hmm... I understand the logic, and can see that it works. However, it's a bit odd to have to scroll vertically to see the entire sentence (yes, my current screen only has about 200 columns or so), and it doesn't make for efficient reading.
On the other hand it doesn't matter for the generated text which most people read.

So my conclusion is that we might go for this, but it needs more thought. And even if we were to change it, I'm also reluctant to apply a global command to change it, since there's a risk that it messes up something else; so if we go for it I would say apply it gradually.

Reply:
I'd also be OK with doing it gradually.  Ideally this should be divided up into commits that only convert a manageable piece into sentence-based line breaking, and commits that make actual content changes.  If we get into the habit of doing it this way, I think it would be a relatively painless process (but slow, of course).